app/src/main/java/com/github/orkest/ui/DeezerWelcomeActivity.kt

```diff
@@ -50,7 +51,16 @@ class DeezerWelcomeActivity : AppCompatActivity(){
                         if(data.access_token != "" && data.access_token != null){
                             val userIdFuture = DeezerApiIntegration().fetchTheUserIdInTheDeezerDatabase(data.access_token!!)
                             userIdFuture.thenAccept {
-                                user ->
+                                user -> val playlistIdFuture = DeezerApiIntegration().createANewPlaylistOnTheUserProfile(user.id,data.access_token!!)
+                                playlistIdFuture.thenAccept {
+                                    playlistId ->
+                                    if (playlistId != "" && playlistId !=null){
+                                    db.storeDeezerInformationsInDatabase(Constants.CURRENT_LOGGED_USER,data.access_token!!, user.id,playlistId)
+                                    }
+                                    else {
+                                        Log.d("DEEZER_OAUTH_FAIL", "Failed to store deezer informations")
+                                    }
+                                }
                             }
 
                         }
```

Comment:
You should have better variable names. For example "code" and "hello" are too vague. 

You should also put an error handling mechanism if the val future fails.

You can write playlistId.isNotBlank() instead of != "".

Reply:
1) I will change the variable names.
2)The error mechanism will be implemented in the future, because it is not a trivial task 
3)Thank you for the tip